tensorflow/lite/micro/kernels/ceva/fully_connected.cc

```diff
@@ -0,0 +1,257 @@
+                                   const TfLiteEvalTensor* input,
+                                   const TfLiteEvalTensor* filter,
+                                   const TfLiteEvalTensor* bias,
+                                   TfLiteEvalTensor* output) {
+  tflite::FullyConnectedParams op_params = FullyConnectedParamsQuantized(data);
+
+  int input_shape_DimensionsCount =
+      tflite::micro::GetTensorShape(input).DimensionsCount();
+  int weights_shape_DimensionsCount =
+      tflite::micro::GetTensorShape(filter).DimensionsCount();
+  int* weights_shape_DimsData =
+      const_cast<int*>(tflite::micro::GetTensorShape(filter).DimsData());
+  int bias_shape_DimensionsCount =
+      tflite::micro::GetTensorShape(bias).DimensionsCount();
+  int output_shape_DimensionsCount =
+      tflite::micro::GetTensorShape(output).DimensionsCount();
+  int* output_shape_DimsData =
+      const_cast<int*>(tflite::micro::GetTensorShape(output).DimsData());
+
+  void* params = (void*)&op_params;
+  int8_t* inputp =
+      const_cast<int8_t*>(tflite::micro::GetTensorData<int8_t>(input));
+  int8_t* filterp =
+      const_cast<int8_t*>(tflite::micro::GetTensorData<int8_t>(filter));
+  int32_t* biasp =
+      const_cast<int32_t*>(tflite::micro::GetTensorData<int32_t>(bias));
+  int8_t* outputp =
+      const_cast<int8_t*>(tflite::micro::GetTensorData<int8_t>(output));
+
+#ifdef MCPS_MEASUREMENT
+  int batches = output_shape_DimsData[0];
+  int output_depth = weights_shape_DimsData[weights_shape_DimensionsCount - 2];
+  int accum_depth = weights_shape_DimsData[weights_shape_DimensionsCount - 1];
+  MCPS_START_ONE;
+#endif
+
+  int sizeof_scr = output_shape_DimsData[1];
+
+  if (sizeof_scr > CEVA_TFLM_KERNELS_SCRATCH_SIZE_VAL) {
+    TF_LITE_KERNEL_LOG(context, "Scratch size (%d) less that required (%d)",
+                       CEVA_TFLM_KERNELS_SCRATCH_SIZE_VAL, sizeof_scr);
+    return kTfLiteError;
+  }
+
+  CEVA_TFLM_FullyConnected_int8(
+      params,
+      input_shape_DimensionsCount,  // GetTensorShape(input),
+      inputp,
+      weights_shape_DimensionsCount,  // GetTensorShape(filter),
+      weights_shape_DimsData, filterp,
+      bias_shape_DimensionsCount,  // GetTensorShape(bias),
+      biasp,
+      output_shape_DimensionsCount,  // GetTensorShape(output),
+      output_shape_DimsData, outputp, CEVA_TFLM_KERNELS_SCRATCH);
+#ifdef MCPS_MEASUREMENT
+  MCPS_STOP_ONE(
+      "Test params:Call CEVA_TFLM_FullyConnected_int8 inetrnal loop = %dx%dx%d",
+      batches, output_depth, accum_depth);
+#endif
+
+  return kTfLiteOk;
+}
+
+TfLiteStatus EvalFloatCEVA(TfLiteContext* context, TfLiteNode* node,
+                           TfLiteFusedActivation activation,
+                           const TfLiteEvalTensor* input,
+                           const TfLiteEvalTensor* filter,
+                           const TfLiteEvalTensor* bias,
+                           TfLiteEvalTensor* output) {
+  float output_activation_min, output_activation_max;

```

Comment:
avoid these local variables, directly pass in `op_param.float_activation_min` as the output params.